app-dashboard/js/view/map.js

```diff
@@ -213,9 +225,47 @@ define([
                     format: 'image/png',
                     transparent: true,
                     srs: 'EPSG:4326',
-                    cql_filter: `id_code=${region_id}`
+                    cql_filter: `id_code=${region_id}`,
                 });
-            this.region_layer.addTo(this.map);
+            this.region_layer.setZIndex(20);
+            this.addOverlayLayer(this.region_layer, 'Administrative Boundary');
+            // TODO: Enable this line to show exposed buildings when the layer is available
+            // TODO: issue: https://github.com/kartoza/fbf-project/issues/123
+            // this.showExposedBuildings(region, region_id);
+        },
+        showExposedBuildings: function (region, region_id) {
+            const that = this;
+            if(this.exposed_layers){
+                this.exposed_layers.forEach(l => that.removeOverlayLayer(l.layer));
+                this.exposed_layers = null;
+            }
+            dispatcher.trigger('map:redraw');
+
+            let id_key = {
+                'district': 'dc_code',
+                'sub_district': 'sub_district_id',
+                'village': 'village_id',
+            }
+
+            this.exposed_layers = this.depth_class_collection.map(function (depth_class) {
+                let label = `Exposed Buildings in Depth Class: ${depth_class.get('label')}`
+                let exposed_layer = L.tileLayer.wms(
+                    geoserverUrl,
+                    {
+                        layers: `kartoza:exposed_buildings`,
+                        format: 'image/png',
```

Comment:
I wonder if we can get a performance / size improvement by using GIF?

Reply:
We can easily switch later to try. I've setup most WMS request with easier configuration like this.